examples/tensorflow-comparison/.saturn/saturn.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "example-tensorflow-comparison",
+  "image_uri": "public.ecr.aws/saturncloud/saturn-tensorflow:2022.01.06",
+  "description": "Training single node and multi GPU models in TensorFlow and drawing comparison",
+  "working_directory": "/home/jovyan/examples/examples/tensorflow-comparison",
+  "extra_packages": {
+    "pip": "wandb"
+  },
+  "start_script": "wandb login --relogin $WANDB_LOGIN || true",
+  "git_repositories": [
+    {
+      "url": "https://github.com/saturncloud/examples"
+    }
+  ],
+  "jupyter_server": {
+    "disk_space": "100Gi",
+    "instance_type": "8xlarge"
```

Comment:
I don't think this is the right instance type for GPU. It should use the instance type that has GPU available for hosted